dubbo-registry/dubbo-registry-consul/src/main/java/org/apache/dubbo/registry/consul/ConsulRegistry.java

```diff
@@ -154,6 +156,24 @@ public void doUnsubscribe(URL url, NotifyListener listener) {
         notifier.stop();
     }
 
+    @Override
```

Comment:
What is the purpose of the change? I could not find any explanation in your pr description.

Reply:
If we use lookUp method from parent class: AbstractRegistry, we cannot get our immediate provider list. I choose to implement this method like what we do in https://github.com/apache/incubator-dubbo/blob/9ce0676908adbb5d073727c47b5a32dda153230e/dubbo-registry/dubbo-registry-zookeeper/src/main/java/org/apache/dubbo/registry/zookeeper/ZookeeperRegistry.java#L195 to keep our public method  with consistent meaning. And you can check it in ConsulRegistryTest.testLookup()